### PR TITLE
adding explicit kubeconfig

### PR DESCRIPTION
**Pull request by elluck91**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -25,14 +25,21 @@ inputs:
...
+    description: 'AWS_ACCESS_KEY_ID'
+    required: true
+  aws_secret_access_key:
+    description: 'AWS_SECRET_ACCESS_KEY'
+    required: true
+  aws_default_region:
+    description: 'AWS_DEFAULT_REGION'
+    required: true
+  aws_org_id:
+    description: 'AWS_ORG_ID'
```
Comment (aalpert-bluescape) [inline action.yml:38]: Wouldn't this stop them from being secret?

Comment (yokiworks) [reply action.yml:38]: yes I think they need to be provided as env vars as before

File: entrypoint.sh [hunk 2]
```diff
@@ -8,17 +8,23 @@ CLUSTER=${5}
...
 git clone https://${GITHUB_PAT}@github.com/${ORG}/${INFRA_REPO}.git
 cd infrastructure
 
+echo ${AWS_DEFAULT_REGION}
+echo ${INPUT_AWS_ACCESS_KEY_ID}
 aws configure set region ${AWS_DEFAULT_REGION}
 aws configure set aws_secret_access_key ${AWS_SECRET_ACCESS_KEY}
 aws configure set aws_access_key_id ${AWS_ACCESS_KEY_ID}
-aws configure set role_arn "arn:aws:iam::${AWS_ACCOUNT_ID}:role/adminAssumeRole"
+aws configure set role_arn "arn:aws:iam::${AWS_ORG_ID}:role/adminAssumeRole"
```
Comment (yokiworks) [inline entrypoint.sh:25]: why you rename account to org?

File: action.yml (hunk 1 above)
Comment (elluck91) [reply action.yml:38]: actions don't support env variables - this is the only way.

File: entrypoint.sh (hunk 2 above)
Comment (elluck91) [reply entrypoint.sh:25]: this refers to the organization in the account, not the account itself.

Comment (aalpert-bluescape) [reply entrypoint.sh:25]: No, I think the way AWS organization nomenclature works is that an organization is a grouping of accounts with a master account, but the IAM resources are per account (and org wide settings are those on the master account).

Not a big deal though.

**Review by yokiworks — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (aalpert-bluescape) [reply action.yml:38]: Further discussion shows they are still secret even when accessed in this manner

**Review by aalpert-bluescape — APPROVED** [review]

**Pull request merged**